redlines/__init__.py

```
@@ -65,7 +64,18 @@
 
 """
 
+from . import utils
+from .utils import ConversionManager, Styles
 from .document import *
```

Ruff (0.13.1)

`from .document import *` used; unable to detect undefined names

(F403)